common/actions/initialize/main.js

```diff
@@ -1,16 +1,19 @@
-const utils = require('../../../utils');
+const { getInput, execStep, setFailed } = require('../../../utils');
 
 try{
-    const ghToken = utils.getInput('gh-token');
+    const npmNexusAuth = getInput('nexus-token');
 
-    utils.execStep(
-        `npm config set //npm.pkg.github.com/:_authToken ${ghToken}`,
+    execStep(
+        [
+            `npm config set //repositories-proxy.mrf.io/nexus/repository/npm-internal/:_authToken ${npmNexusAuth}`,
```

Comment:
Since the proxied registry does not need auth, does this mean we could apply this? @miquelmasrieramrf @davidmohringmrf 

```suggestion
            `npm config set //repositories-proxy.mrf.io/nexus/repository/npm-internal`,
```